src/Core/GCStashList.cpp

```diff
@@ -336,7 +333,6 @@ void GCStashList::setStashItem(BYTE rack, BYTE index, Item* pItem)
 {
 	__BEGIN_TRY
 
-#ifdef	__GAME_SERVER__
```

Comment:
Here, the `__GAME_SERVER__` macro is not defined when building this file, so the code is ignored